TeamTracker/app/src/main/java/com/example/teamtracker/activities/LoginActivity.java

```diff
@@ -177,7 +177,7 @@ public void onErrorResponse(VolleyError error) {
                         }
                     });
                     thread.start();
-                }
+                }afsdfasdfa
```

Comment:
what type of demo feature is this?

Reply:
I used this to help one of our friends for teaching git.